format/spec.md

```diff
@@ -408,16 +406,17 @@ When `null`, a row's `_row_id` field is assigned to the `first_row_id` from its
 
 Values for `_row_id` and `_last_updated_sequence_number` are either read from the data file or assigned at read time. As a result on read, rows in a table always have non-null values for these fields when lineage is enabled.
 
-When an existing row is moved to a different data file for any reason, writers are required to write `_row_id` and `_last_updated_sequence_number` according to the following rules:
+When an existing row is moved to a different data file for any reason, writers should write `_row_id` and `_last_updated_sequence_number` according to the following rules:
 
 1. The row's existing non-null `_row_id` must be copied into the new data file
 2. If the write has modified the row, the `_last_updated_sequence_number` field must be set to `null` (so that the modification's sequence number replaces the current value)
 3. If the write has not modified the row, the existing non-null `_last_updated_sequence_number` value must be copied to the new data file
 
+Engines may model operations as deleting rows and inserting rows or as modifications to rows that preserve row ids.
```

Comment:
I think we'd still need to call out somewhere that "_last_updated_sequence_number" should be preserved if a row is unmodified in the case an operation is not modeled as a delete + insert, and an exisitng row is moved to a different file. 

The section above on line 391 used to say that, but I think now it's removed.

Reply:
Isn't that what line 413 states?

Reply:
Ah sorry about that, I was only looking at the changed lines and missed that this already is there. Cool!

Reply:
I think a comma will suffice.  I don't think comma or semi-colon would be correct here.  I might be able to get away with removing the other conjunction by using "deleting/inserting rows", which would avoid the whole issue.